packages/core/test/spawn.types.test.ts

```diff
@@ -0,0 +1,49 @@
+import { ActorRefFrom, assign, createMachine } from '../src';
+
+describe('spawn inside machine', () => {
+  it('input is required when defined in actor', () => {
```

Comment:
we don't really check that it's required here, u just pass an input as an argument but the same test would pass without the patch introduced by this PR

Reply:
Should this be inverted with a `@ts-expect-error` instead? Or is there another better way to test types?